distro/pom.xml

```diff
@@ -75,9 +76,14 @@
         </dependency>
         <dependency>
             <groupId>org.bahmni.module</groupId>
-            <artifactId>bahmnicore-omod</artifactId>
+            <artifactId>bahmnicore-ts-omod</artifactId>
```

Comment:
am assuming this is temporary

Reply:
yes @angshu... while merging with `master` branch, will revert back this change.